content/blogposts/2024/sourcegraph-release-may-2024.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Sourcegraph May 2024 release"
+publishDate: 2024-05-09T10:00-07:00
+description: "The latest Sourcegraph release includes a new point-in-time search filter for Code Search."
+tags: [blog, release]
+slug: "release/May-2024"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/may-2024/sourcegraph-may2024-release-image.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/may-2024/sourcegraph-may2024-release-image.png
+changelogItems:
+  - description: "Improved the performance of language stats insights by 50-70%, significantly reducing pain for customers with large repos. The timeout for language stats insights has also been raised from 3 to 5 minutes. Both are configurable using environment variables on the frontend container."
+    url: https://sourcegraph.com/docs/code_insights/explanations/administration_and_security_of_code_insights#language-stats-performance-configuration
+    category: Code Insights
+  - description: "Improved Zoekt CPU usage and scalability for customers with a large number of repos."
+    url:
+    category: Search
+  - description: "Greatly improved git blame performance on large files, reducing the time it takes for a git blame on a large file to load from several seconds down to less than 500 milliseconds."
+    url:
+    category: Search
+  - description: "Search Jobs now supports diff, commit, and path searches. Before, only file searches were supported."
+    url:
+    category: Source
+ 
+
+
+---
+
+<br/>
+
+### Point-in-time search filter for Code Search
+
+<br/>
+
+Sourcegraph 5.4.0 brings improvements to Code Search with a new search filter [rev:at.time()](https://sourcegraph.com/docs/code-search/queries#structural-search) that provides point-in-time searching. This allows for searching a repo’s state at a specific timestamp rather than at a specific commit, branch, or tag.
+
+
+<Video 
+  source={{
+    mp4: 'blog/release-post/may-2024/rev-at-time'
+  }}
+  loop={true}
+  title="Searching by the new point-in-time search filter"
+/>
+
+
+
+<br/>
+
+### Improved Code Insights chart drill-downs
+
+<br/>
+
+Code Insights chart drill-downs now utilize the new point-in-time search filter to more accurately represent the results that produced a data point on a chart. Previously a diff search was used to show the difference between a selected point in time on a chart and the one before it, but Code Insights can now show a repository’s state at that exact timestamp. This functionality is useful for those searching large repositories or repositories with lots of history.
+
+<Video 
```

Comment:
Possible to show controls for this video? It's long and feels like users should be able to control it instead of looping

Reply:
@jdorfman do you know how to add this? I can't remember the code for showing controls

Reply:
@iskyOS checking

Reply:
@iskyOS 

```
<Video 
  source={{
    mp4: 'blog/shift-left/01-code-reuse'
  }}
  controls={true}
  ...
```